src/components/core/Bookmark/loginmodal.js

```diff
@@ -0,0 +1,74 @@
+/** @jsx jsx */ import { jsx, css } from "@emotion/core";
```

Comment:
Please rename the filename with React files naming convention - should be `LoginModal.js`